On Solaris one needs to include limits.h in order to use PATH_MAX

--- screen-4.5.0/screen.c
+++ screen-4.5.0/screen.c
@@ -44,6 +44,7 @@
 #endif
 #include <ctype.h>
 #include <fcntl.h>
+#include <limits.h>
 
 #ifdef sgi
 # include <sys/sysmacros.h>
